submitit INFO (2025-02-18 11:10:35,400) - Starting with JobEnvironment(job_id=800241, hostname=arbeitsrechner, local_rank=0(1), node=0(1), global_rank=0(1))
submitit INFO (2025-02-18 11:10:35,400) - Loading pickle: /home/s3811141/test/randomtest_4290/omniopt/ax/runs/qader_example/0/single_runs/800241/800241_submitted.pkl
2025-02-18 11:10:37.484469: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2025-02-18 11:10:37.487945: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2025-02-18 11:10:37.498232: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:477] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1739873437.515504  800428 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1739873437.520535  800428 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-02-18 11:10:37.538038: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2025-02-18 11:10:39.875501: E external/local_xla/xla/stream_executor/cuda/cuda_driver.cc:152] failed call to cuInit: INTERNAL: CUDA error: Failed call to cuInit: UNKNOWN ERROR (303)
/home/s3811141/.omniax_test_x86_64_Python_3.10.0/lib/python3.10/site-packages/keras/src/trainers/data_adapters/py_dataset_adapter.py:121: UserWarning: Your `PyDataset` class should call `super().__init__(**kwargs)` in its constructor. `**kwargs` can include `workers`, `use_multiprocessing`, `max_queue_size`. Do not pass these arguments to `fit()`, as they will be ignored.
  self._warn_if_super_not_called()

Parameters: {"lr": 0.04146388256574766, "epochs": 1, "width_and_height": 20}
Debug-Infos: 
========
DEBUG INFOS START:
Program-Code: bash /home/s3811141/repos/OmniOpt/ax/.tests/example_network/run.sh --learning_rate=0.04146388256574766 --epochs=1 --width=20 --height=20
pwd: /home/s3811141/test/randomtest_4290/omniopt/ax
File: /home/s3811141/repos/OmniOpt/ax/.tests/example_network/run.sh
UID: 1000
GID: 1000
Status-Change-Time: 1739348026.874241
Size: 3082 Bytes
Permissions: -rwxrwxr-x
Owner: s3811141
Last access: 1739867959.3379517
Last modification: 1739348026.8702412
Hostname: arbeitsrechner
========
DEBUG INFOS END

bash /home/s3811141/repos/OmniOpt/ax/.tests/example_network/run.sh --learning_rate=0.04146388256574766 --epochs=1 --width=20 --height=20
stdout: Model: "sequential"
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┓
┃ Layer (type)                    ┃ Output Shape           ┃       Param # ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━┩
│ conv2d (Conv2D)                 │ (None, 18, 18, 4)      │           112 │
├─────────────────────────────────┼────────────────────────┼───────────────┤
│ conv2d_1 (Conv2D)               │ (None, 16, 16, 4)      │           148 │
├─────────────────────────────────┼────────────────────────┼───────────────┤
│ flatten (Flatten)               │ (None, 1024)           │             0 │
├─────────────────────────────────┼────────────────────────┼───────────────┤
│ dense (Dense)                   │ (None, 32)             │        32,800 │
├─────────────────────────────────┼────────────────────────┼───────────────┤
│ dense_1 (Dense)                 │ (None, 32)             │         1,056 │
├─────────────────────────────────┼────────────────────────┼───────────────┤
│ dense_2 (Dense)                 │ (None, 5)              │           165 │
└─────────────────────────────────┴────────────────────────┴───────────────┘
 Total params: 34,281 (133.91 KB)
 Trainable params: 34,281 (133.91 KB)
 Non-trainable params: 0 (0.00 B)
Found 880 images belonging to 5 classes.
Found 218 images belonging to 5 classes.

[1m 1/88[0m [37m━━━━━━━━━━━━━━━━━━━━[0m [1m1:37[0m 1s/step - accuracy: 0.2000 - loss: 1.7479
[1m 6/88[0m [32m━[0m[37m━━━━━━━━━━━━━━━━━━━[0m [1m0s[0m 10ms/step - accuracy: 0.1347 - loss: 4.0664
[1m11/88[0m [32m━━[0m[37m━━━━━━━━━━━━━━━━━━[0m [1m0s[0m 11ms/step - accuracy: 0.1312 - loss: 3.4900
[1m16/88[0m [32m━━━[0m[37m━━━━━━━━━━━━━━━━━[0m [1m0s[0m 11ms/step - accuracy: 0.1543 - loss: 3.1290
[1m21/88[0m [32m━━━━[0m[37m━━━━━━━━━━━━━━━━[0m [1m0s[0m 11ms/step - accuracy: 0.1748 - loss: 2.8971
[1m26/88[0m [32m━━━━━[0m[37m━━━━━━━━━━━━━━━[0m [1m0s[0m 11ms/step - accuracy: 0.1908 - loss: 2.7300
[1m31/88[0m [32m━━━━━━━[0m[37m━━━━━━━━━━━━━[0m [1m0s[0m 11ms/step - accuracy: 0.2015 - loss: 2.6050
[1m35/88[0m [32m━━━━━━━[0m[37m━━━━━━━━━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2077 - loss: 2.5246
[1m40/88[0m [32m━━━━━━━━━[0m[37m━━━━━━━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2140 - loss: 2.4418
[1m45/88[0m [32m━━━━━━━━━━[0m[37m━━━━━━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2191 - loss: 2.3728
[1m50/88[0m [32m━━━━━━━━━━━[0m[37m━━━━━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2242 - loss: 2.3138
[1m55/88[0m [32m━━━━━━━━━━━━[0m[37m━━━━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2288 - loss: 2.2637
[1m60/88[0m [32m━━━━━━━━━━━━━[0m[37m━━━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2323 - loss: 2.2210
[1m65/88[0m [32m━━━━━━━━━━━━━━[0m[37m━━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2348 - loss: 2.1839
[1m70/88[0m [32m━━━━━━━━━━━━━━━[0m[37m━━━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2367 - loss: 2.1510
[1m75/88[0m [32m━━━━━━━━━━━━━━━━━[0m[37m━━━[0m [1m0s[0m 12ms/step - accuracy: 0.2383 - loss: 2.1218
[1m80/88[0m [32m━━━━━━━━━━━━━━━━━━[0m[37m━━[0m [1m0s[0m 12ms/step - accuracy: 0.2397 - loss: 2.0954
[1m85/88[0m [32m━━━━━━━━━━━━━━━━━━━[0m[37m━[0m [1m0s[0m 12ms/step - accuracy: 0.2412 - loss: 2.0714
[1m88/88[0m [32m━━━━━━━━━━━━━━━━━━━━[0m[37m[0m [1m3s[0m 16ms/step - accuracy: 0.2425 - loss: 2.0537 - val_accuracy: 0.2798 - val_loss: 1.5526
RESULT: 1.675902
LOSS: 1.675902
VAL_ACCURACY: 0.2798
VAL_LOSS: 1.5526
RAM_USAGE: 635.64
RUNTIME: 7

stderr: 2025-02-18 11:10:37.484469: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2025-02-18 11:10:37.487945: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2025-02-18 11:10:37.498232: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:477] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1739873437.515504  800428 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1739873437.520535  800428 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-02-18 11:10:37.538038: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2025-02-18 11:10:39.875501: E external/local_xla/xla/stream_executor/cuda/cuda_driver.cc:152] failed call to cuInit: INTERNAL: CUDA error: Failed call to cuInit: UNKNOWN ERROR (303)
/home/s3811141/.omniax_test_x86_64_Python_3.10.0/lib/python3.10/site-packages/keras/src/trainers/data_adapters/py_dataset_adapter.py:121: UserWarning: Your `PyDataset` class should call `super().__init__(**kwargs)` in its constructor. `**kwargs` can include `workers`, `use_multiprocessing`, `max_queue_size`. Do not pass these arguments to `fit()`, as they will be ignored.
  self._warn_if_super_not_called()

Result: {'VAL_ACCURACY': 0.2798, 'RAM_USAGE': 635.64}
EXIT_CODE: 0
submitit INFO (2025-02-18 11:10:49,013) - Job completed successfully
submitit INFO (2025-02-18 11:10:49,014) - Exiting after successful completion
